docs/HOW_WE_WORK.md

```diff
@@ -25,3 +33,7 @@ We track our work Kanban style in a [Zenhub board](https://app.zenhub.com/worksp
 - Responsibilities:
   - Project Maintainer and/or Project Architect - Review issues on Inbox, break them down if necessary, move them into Ready when it is the right time. Also, label issues with priority and difficulty.
   - Contributors move issues between the Ready, In Progress and Review/QA Colums. Use help wanted and blocked labels in case they want to flag that work.
+  
+## Contributing Guildeines 
```

Comment:
This is already part of the main entry README.md - not sure we should copy-paste a link to it in more places.

Reply:
It makes sense as the README is designed for both users and contributors and this doc is designed for contributors (it reminds contributors that the guildelines are part of how we work).